Stripe/PublicHeaders/ApplePay/STPAPIClient+ApplePay.h

```diff
@@ -21,6 +21,9 @@
  */
 - (void)createTokenWithPayment:(nonnull PKPayment *)payment completion:(nonnull STPTokenCompletionBlock)completion;
 
+// Encodes the data from an ABRecordRef (name/address) into a string. This method is used internally by STPAPIClient; you should not use it in your own code.
```

Comment:
No need to expose this in the header (I believe `formEncodedDataForPayment` was exposed for testing).